### PR TITLE
Feature/regional-routes-descriptions

### PR DESCRIPTION
**Pull request by juhomakkonen**
# Add descriptions for regional routes.

### Cards 114 and 116

-----------------------------------------------------------------------------------------------
#### Add descriptions for regional routes
 1. src/views/MobilitySettingsView/components/RouteLength/RouteLength.js
     * Add cases for each regional route to get correct translation.
   
 2. src/i18n/en.js & src/i18n/fi.js & src/i18n/sv.js
     * Update translations
     
#### Add Scale-up text into README
 1. README.md
     * Mention Scale-up project.

#### Adjust icon change level
 2. src/components/MobilityPlatform/BicycleStands/BicycleStands.js
     * Icons will now change at the same time as shared use cars' icons change.

### CODE REVIEW

File: src/views/MobilitySettingsView/components/RouteLength/RouteLength.js [hunk 1]
```diff
@@ -28,6 +28,78 @@ const RouteLength = ({ classes, intl, route }) => {
             {intl.formatMessage({ id: 'mobilityPlatform.menu.bicycleRoutes.auraRiverTrail' })}
           </Typography>
         );
+      case 'Seutureitti 1':
```
Comment (aceViilee) [inline src/views/MobilitySettingsView/components/RouteLength/RouteLength.js:31]: Does some crap like this work?
let text = "";
for (let i = 1; i < 13; i++) {
      return (
          <Typography component="p" variant="body2" className={classes.margin}>
            {intl.formatMessage({ id: 'mobilityPlatform.menu.bicycleRoutes.regionalTrail'+i })}
          </Typography>
        );
}

**Review by aceViilee — COMMENTED** [review]
Some adjustments if suggestion works

File: src/views/MobilitySettingsView/components/RouteLength/RouteLength.js (hunk 1 above)
Comment (juhomakkonen) [reply src/views/MobilitySettingsView/components/RouteLength/RouteLength.js:31]: I'll look into it.

Comment (juhomakkonen) [reply src/views/MobilitySettingsView/components/RouteLength/RouteLength.js]: I refactored it to reduce repetition.

**Review by aceViilee — APPROVED** [review]

**Pull request merged**